osu/chapter.md

```diff
@@ -1 +1,468 @@
+
+For this part, we will sometimes compare the interfaces of the current software and its legacy version. We will call the current game that we have analyzed so far "osu!lazer", while the old version will be called "legacy osu!".
+
+### Touch points
+
+First, let's review the touch points of osu!lazer, ie all the places where the user interacts with osu!lazer. The following graph shows how the user jumps from one touch point to another. The peripheral used for each is mentioned. The term "pointing device" is grouping both mouse and drawing tablet, as many players are using a tablet to be able to aim the targets using absolute positioning. Keyboard-oriented interactions are in blue, while pointing-oriented interactions are in pink. Logically, touch points relying on both peripherals are in violet.
+
+![Touch points](./touch_points.svg)
+<br> **Figure 10** - _Touch points in osu!'s interface_
+
+
+This graph shows that osu! ( both osu!lazer and legacy osu!) can be played with usual PC peripherals.
+
+### Interface
+
+"A user interface is like a joke. If you have to explain it, it’s not good."
+Following that motto, osu!lazer's interface is straightforward, especially considering how close to other rhythm games interfaces it looks like.
+
+From left to right, top to bottom, here are the beatmap selection screens for each of the following games: Hatsune Miku Project Diva, Beatmania, Guitar Hero, and the current in-development osu!lazer.
+
+*Beatmania being an arcade game, the picture comes from a photograph. We couldn't find any better pictures and apologize for this.*
+
+
+|   |   |
+|:-----:|:-----:|
+| ![Project Diva](./HMPDX.jpg) | ![Guitar Hero](./GH.jpg) |
+| Hatsune Miku Project Diva X (Console)| Guitar Hero Live (Consoles)|
+| ![Beatmania Arcade](./BeatmaniaArcade.jpg)  |  ![osu!lazer](./osulazer.jpg)   |
+| Beatmania IIDX (Arcade)| osu!lazer (PC) |
+
+We can notice several similarities.
+
+* All games have a list of beatmaps (although the term "beatmap" is osu!-specific, the concept is the same).
+* They are arranged into a vertical list.
+* The difficulty is rated with a number. Most games are using a star-counting system. Beatmania is an exception which only uses numbers.
+* Some art about the current highlighted beatmap is shown on the side.
+* Also on the side, the current high scores are displayed. 
+* All games have a sorting function. Still, this feature is more developed for osu! for two major reasons:
+    * First, you can download as many beatmaps as you want, unlike the other games who have a limited number of beatmaps,
+    * Being on PC, you can use the keyboard to search, which is impossible on other supports.
```

Comment:
"impossible on other supports"

Do you mean "platforms"? I'm not sure what you mean with "supports", and it is the only occurrence of this word in your chapter so I can't infer it from context either.

Reply:
You're right, it's other platforms. Fixed.